### DISCUSSION TITLE
Error: ModuleNotFoundError: No module named 'pytz'

### DISCUSSION THREAD

**Discussion by hidp123** [skyfielders/python-skyfield#765] [Q&A] [answered]
I have installed Python 3.10.5 and installed skyfield via `pip install skyfield`. Now, when I run the following code in Visual Studio Code from the [examples](https://rhodesmill.org/skyfield/examples.html#what-time-is-solar-noon-when-the-sun-transits-the-meridian) page I get the following error:

ModuleNotFoundError: No module named 'pytz'
and
NameError: name 'timezone' is not defined

The .py file is saved on the desktop. Any idea what is missing and why?

Thanks

**Comment by davidmikolas** [comment]
Hi @hidp123

Can you copy/paste the entire error message rather than just excerpts? There may be additional information like version numbers, line numbers and other helpful bits.

Thanks!

**Reply by hidp123** [reply]
I managed to fix the issue by installing pytz via `pip install pytz`.
But I was surprised I needed to do it manually!

**Comment by hidp123** [comment, marked as answer]
I managed to fix the issue by installing pytz via `pip install pytz` in PowerShell.
